code/modules/mob/observer/freelook/marker/biomass.dm

```diff
@@ -11,7 +11,7 @@
 
 /datum/biomass_source/New(var/datum/_source = null, var/datum/_target = null, var/total_mass = 0, var/duration = 1 SECOND)
 	.=..()
-	source = "\ref[_source]"
+	source = "\ref[_source]" //I have no idea what's going on with this, but I'm too scared to change it. Please review this change if you know why it was implemented this way.
```

Comment:
this is just a weak pointer, to hold a reference to an object without messing with its GC

Reply:
...Weird.

So the biomass source can still continue with a nullpointer if the object is deleted?

Reply:
Well in some cases, it depends on the specific source. Some (like absorbing a human corpse) will delete themselves if they find their source atom is gone. While others (like reclaiming dead necromorphs) will just not do certain behaviours and continue working as normal if their source atom is deleted.

The latter case is to prevent people gibbing corpses (and in the cases of exploders which gib themselves)